includes/storage/SMW_SQLHelpers.php

```diff
@@ -120,7 +122,7 @@ private static function createTable( $tableName, array $fields, $db ) {
 	 * @param DatabaseBase|Database $db
 	 * @param object $reportTo Object to report back to.
 	 */
-	private static function updateTable( $tableName, array $fields, $db, $reportTo ) {
+	private static function updateTable( $tableName, array $fields, $tableOptions = array(), $db, $reportTo ) {
```

Comment:
Just need this for https://github.com/SemanticMediaWiki/SemanticMediaWiki/commit/a8312dc0f62cb9dc8b4ae0791f1bdc9748b8a303.